### PR TITLE
Updating the transport config to match with solana endpoint

### PR DESCRIPTION
**Pull request by godmodegalactus**

### CODE REVIEW

File: services/src/quic_connection_utils.rs [hunk 1]
```diff
@@ -79,6 +79,18 @@ impl QuicConnectionUtils {
...
+
+        const DATAGRAM_RECEIVE_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
+        const DATAGRAM_SEND_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
+        const INITIAL_MAXIMUM_TRANSMISSION_UNIT: u16 = MINIMUM_MAXIMUM_TRANSMISSION_UNIT;
+        const MINIMUM_MAXIMUM_TRANSMISSION_UNIT: u16 = 1280;
+        transport_config.datagram_receive_buffer_size(Some(DATAGRAM_RECEIVE_BUFFER_SIZE));
+        transport_config.datagram_send_buffer_size(DATAGRAM_SEND_BUFFER_SIZE);
+        transport_config.initial_mtu(INITIAL_MAXIMUM_TRANSMISSION_UNIT);
+        transport_config.max_concurrent_bidi_streams(VarInt::from(0u8));
+        transport_config.max_concurrent_uni_streams(VarInt::from(0u8));
```
Comment (grooviegermanikus) [inline services/src/quic_connection_utils.rs:91]: are you sure that does work for us?

File: services/src/quic_connection_utils.rs [hunk 2]
```diff
@@ -79,6 +79,18 @@ impl QuicConnectionUtils {
         let timeout = IdleTimeout::try_from(Duration::from_secs(1)).unwrap();
         transport_config.max_idle_timeout(Some(timeout));
         transport_config.keep_alive_interval(Some(Duration::from_millis(500)));
+
+        const DATAGRAM_RECEIVE_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
```
Comment (grooviegermanikus) [inline services/src/quic_connection_utils.rs:83]: that's one 1GB !

File: services/src/quic_connection_utils.rs [hunk 3]
```diff
@@ -79,6 +79,18 @@ impl QuicConnectionUtils {
...
+        const DATAGRAM_SEND_BUFFER_SIZE: usize = 1024 * 1024 * 1024;
+        const INITIAL_MAXIMUM_TRANSMISSION_UNIT: u16 = MINIMUM_MAXIMUM_TRANSMISSION_UNIT;
+        const MINIMUM_MAXIMUM_TRANSMISSION_UNIT: u16 = 1280;
+        transport_config.datagram_receive_buffer_size(Some(DATAGRAM_RECEIVE_BUFFER_SIZE));
+        transport_config.datagram_send_buffer_size(DATAGRAM_SEND_BUFFER_SIZE);
+        transport_config.initial_mtu(INITIAL_MAXIMUM_TRANSMISSION_UNIT);
```
Comment (grooviegermanikus) [inline services/src/quic_connection_utils.rs:89]: not sure if setting the initial value will prevent that heuristic will move the packet size up.
**would expect to have a max (or flex) size**

**Review by grooviegermanikus — CHANGES_REQUESTED** [review]

File: services/src/quic_connection_utils.rs (hunk 3 above)
Comment (grooviegermanikus) [reply services/src/quic_connection_utils.rs:89]: this?
image

File: services/src/quic_connection_utils.rs (hunk 1 above)
Comment (godmodegalactus) [reply services/src/quic_connection_utils.rs:91]: should work

File: services/src/quic_connection_utils.rs (hunk 2 above)
Comment (godmodegalactus) [reply services/src/quic_connection_utils.rs:83]: reduced to 64 MB

**Review by grooviegermanikus — APPROVED** [review]
lgtm

**Pull request merged**